### ISSUE TITLE
[feature request] Sync files by pattern

### ISSUE THREAD

**Issue by tony19** [vuejs-translations/ryu-cho#7]
Currently, `ryo-cho` only tracks commits to a specified directory (via `path-starts-with`), but it would be nice to track multiple paths. This would come in handy when tracking both `./docs/` and the Markdown in the root of the project (i.e.,  `./CONTRIBUTING.md` and `./README.md`).

Perhaps we should add an `includes` configuration to `ryo-cho` that accepts an array of glob patterns.

**Comment by kiaking** [comment]
Good idea! I think we can rename `path-starts-with` to `paths` and accept array of paths. I think I'll not be able to tackle this soon but PR is welcome!

**Comment by tony19** [comment]
@kiaking See #8
